releasenotes/notes/fix-unicode-decode-errors-pylons-52d28b22f82e184a.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    pylons: This fix resolves an issue where str.decode caused critical unicode decode errors.
```

This is only for when appsec is enabled right? so should we say this is an appsec bug?

```suggestion
    pylons: This fix resolves an issue where ``str.decode`` could cause critical unicode decode errors when ASM is enabled. ASM is disabled by default.
```

wdyt?